styles.css

```diff
@@ -41,68 +31,212 @@
   flex-direction:column;
   justify-content: space-around;
   align-items:center;
+  position:relative;
   margin: 30px;
   width: 800px;
```

Comment:
a fixed width of 800px is a problem.
we write mobile first, mobile displays are smaller.
you could use % or min-width/max-width for responsiveness.